tests/travis/install.sh

```diff
@@ -1,30 +1,47 @@
 #!/usr/bin/env bash
 
 # NAME
-#     install.sh - Install Travis CI dependencies
+#     install.sh - Create the test fixture.
 #
 # SYNOPSIS
 #     install.sh
 #
 # DESCRIPTION
-#     Creates the test fixture.
+#     Creates the test fixture and places the SUT.
 
-cd "$(dirname "$0")"; source _includes.sh
+cd "$(dirname "$0")" || exit; source _includes.sh
```

Comment:
This all seems _really_ custom. @TravisCarden and/or @kdruckman, can we go over this a bit so I understand what we're doing? Alternatively, a comment would help here.

Reply:
This customization is because we needed to redefine all the orca jobs in order to set our own version variables. Even though the current recommended version is really D9, Orca still has to have it hardcoded as D8.9 in order to support D8.8. So we need to override all of it since there is not yet a way to just override CURRENT_RECOMMENDED, etc.